qbittorrent/umbrel-app.yml

```diff
@@ -0,0 +1,30 @@
+manifestVersion: 1
+id: qbittorrent
+category: Networking
+name: qBittorrent
+version: "4.5.2"
+tagline: A fast, easy and free BitTorrent client
+description: >-
+  The qBittorrent project aims to provide an open-source software alternative to µTorrent.
+  
+  Additionally, qBittorrent runs and provides the same features on all major platforms (FreeBSD, Linux, macOS, OS/2, Windows).
+  
+  Bittorrent is based on the Qt toolkit and libtorrent-rasterbar library.
+  
+  ⚠️ qBittorrent downloads torrents over the Clearnet, not Tor.
+developer: qBittorrent
+website: https://qbittorrent.org/
+dependencies: []
+repo: https://github.com/qBittorrent/qBittorrent
+support: https://github.com/qBittorrent/qBittorrent/issues
+port: 8080
```

Comment:
Should not it be 8094?

Reply:
It works fine like this but I'll try changing it to 8094 now

Reply:
Nope, when I change the port to 8094 it has this error on my Umbrel server: `ERROR: for server  Cannot start service server: driver failed programming external connectivity on endpoint qbittorrent_server_1 (0b6c65cd2850906a63b75de1de38e040d5b0a02561ae0227a4fd3949601fb2a5): Bind for 0.0.0.0:8094 failed: port is already allocated`

It works fine when the port is 8080 so I'll just leave it for now